pages/api/hello.js

```diff
@@ -1,6 +1,12 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+const pool = require('./mypool')
 
 export default (req, res) => {
-  res.statusCode = 200
-  res.json({ name: 'John Doe' })
+  pool.query('SELECT * FROM player ORDER BY id ASC')
+    .then(results => {
+      res.status(200).send(results.rows);
```

Comment:
use `.json` instead of `.send`. Refer this example: https://github.com/vercel/next.js/blob/canary/examples/api-routes/pages/api/people/index.js